lib/utils.ts

```diff
@@ -88,12 +88,31 @@ export const navLinks = [
   { title: "People", path: "/people" },
   { title: "Projects", path: "/projects" },
   { title: "Releases", path: "/releases" },
+  { title: "Discussion", path: "/discussion" },
 ];
 
 export const formatDate = (date: Date) => {
   return format(date, "MMM dd, yyyy");
 };
-type Features = "Projects" | "Releases";
+type Features = "Projects" | "Releases" | "Discussions";
 export const featureIsEnabled = (feature: Features) => {
   return env.NEXT_PUBLIC_FEATURES?.split(",").includes(feature);
 };
+
+export const parseOrgRepoFromURL = (url: string) => {
+  const parts = url.split("/");
+
+  let org, repository;
+
+  if (parts[3] === "orgs") {
+    // Handle the URL format: /orgs/{org}/discussions/{id}
+    org = parts[4];
+    repository = "";
```

Comment:
It's a good practice to use `null` instead of empty string for when any information is not available.